packages/perseus/src/widgets/label-image/answer-pill.tsx

```diff
@@ -58,7 +58,7 @@ export const AnswerPill = (props: {
                 <Pill
                     size="large"
                     kind="accent"
-                    id={id}
+                    id={"perseus-label-image-widget-answer-pill-" + id}
```

Comment:
The coordinates are for the location of the marker, not answer pill. It would be impossible to use the label widget if two markers shared coordinates.

Reply:
I meant the same coordinate in a different `label-image` instance on the same page.  I suspect that's never happened.

Reply:
Ha, I can use `useUniqueId`

Reply:
Fixing this with my next PR